src/components/AstroPiModel/AstroPiModel.scss

```diff
@@ -192,15 +193,15 @@
 
       &-panel {
         &__control {
-          box-shadow: inset 0 0 var(--spacing-2) $rpf-grey-700;
+          box-shadow: inset 0 0 $space-3 $rpf-grey-700;
           background-color: $rpf-grey-700;
```

Comment:
is `space-3` okay (it replaces` spacing-2` - are they equivalent)?

Reply:
Yes, as per the [design system switchover document](https://www.figma.com/file/k8HLf9t06G8QtlMzQlgHGk/RPF-Design-System---Component-Library?type=design&node-id=1816-15724&mode=design&t=N2mcFxl9zLOOxuFY-0) - I think one was based on a 10px grid and the other on a 8px grid or something to that effect, so there's a bit of "give or take" but they're more or less equivalent.